{{cookiecutter.project_dir}}/{{cookiecutter.project_dir}}-ui/src/App.js

```diff
@@ -22,8 +22,10 @@ class App extends React.Component {
 
     translate(event) {
         event.preventDefault();
-        let serviceIp = "http://{{cookiecutter.project_dir}}-api";
-        axios.post(serviceIp + '/api/v1/translate', {
+
+        const { API_HOST, location } = window;
+
+        axios.post(`${location.protocol}//${API_HOST}/api/v1/translate`, {
```

Comment:
This calls the Django rest api and not the express endpoint. The point of the express server is to use run time env instead of build time. It's not proxying the api.

Reply:
We want to show how to use runtime variables without exposing them in the browser. I think we should proxy the api call through it.